aten/src/ATen/native/cudnn/MHA.cpp

```diff
@@ -278,23 +312,29 @@ auto build_graph_and_tensors(
       fe::graph::Tensor_attributes()
           .set_name("Q")
           .set_dim(
-              std::vector<int64_t>(params.q_dim.begin(), params.q_dim.end()))
-          .set_stride(std::vector<int64_t>(
-              params.q_stride.begin(), params.q_stride.end())));
+              std::vector<int64_t>(
+                q.sizes().data(), q.sizes().data() + q.sizes().size()))
```

Comment:
is there some other cleanup to the Params struct that could be done if we are gettin info straight from inputs?

Reply:
That's a good point, probably don't need to pass the `struct` here as it's unused now...
Still need these as fields in the `struct` itself as they are important to avoid collisions in the cache.